### PR TITLE
PWGGA/GammaConv: simplified the trigger mimicking by directly using a…

### PR DESCRIPTION
**Pull request by jokonig**
… build in TRandom3 function instead of GetRandom(). This should be much faster

### CODE REVIEW

File: PWGGA/GammaConvBase/AliConvEventCuts.cxx [hunk 1]
```diff
@@ -3982,11 +3977,7 @@ Bool_t AliConvEventCuts::MimicTrigger(AliVEvent *event, Bool_t isMC ){
       Double_t threshold = thresholdEMCalL1[binRun];
 
       if (isMC && spreadEMCalL1[binRun] != 0.){
-        TF1 triggerSmearing("triggerSmearing","[0]*exp(-0.5*((x-[1])/[2])**2)",0,15);
-        triggerSmearing.SetParameter(0, 1/(spreadEMCalL1[binRun]*TMath::Sqrt(TMath::Pi()*2)));
-        triggerSmearing.SetParameter(1, thresholdEMCalL1[binRun]);
-        triggerSmearing.SetParameter(2, spreadEMCalL1[binRun]);
-        threshold = triggerSmearing.GetRandom();
+        threshold = fRandom.Gaus(thresholdEMCalL0[binRun], spreadEMCalL0[binRun]);
```
Comment (dmuhlhei) [inline PWGGA/GammaConvBase/AliConvEventCuts.cxx]: copy/paste error: L1 here!

File: PWGGA/GammaConvBase/AliConvEventCuts.cxx [hunk 2]
```diff
@@ -4053,11 +4044,7 @@ Bool_t AliConvEventCuts::MimicTrigger(AliVEvent *event, Bool_t isMC ){
       if (binRun==19) return kFALSE;
       Double_t threshold = thresholdEMCalL1G2[binRun];
       if (isMC && spreadEMCalL1G2[binRun] != 0.){
-        TF1 triggerSmearing("triggerSmearing","[0]*exp(-0.5*((x-[1])/[2])**2)",0,15);
-        triggerSmearing.SetParameter(0, 1/(spreadEMCalL1G2[binRun]*TMath::Sqrt(TMath::Pi()*2)));
-        triggerSmearing.SetParameter(1, thresholdEMCalL1G2[binRun]);
-        triggerSmearing.SetParameter(2, spreadEMCalL1G2[binRun]);
-        threshold = triggerSmearing.GetRandom();
+        threshold = fRandom.Gaus(thresholdEMCalL0[binRun], spreadEMCalL0[binRun]);
```
Comment (dmuhlhei) [inline PWGGA/GammaConvBase/AliConvEventCuts.cxx]: L1G2 here!!

File: PWGGA/GammaConvBase/AliConvEventCuts.cxx (hunk 1 above)
Comment (jokonig) [reply PWGGA/GammaConvBase/AliConvEventCuts.cxx]: Sorry that was a stupid mistake... Thanks for checking! It should be corrected now for both

**Comment by dmuhlhei** [comment]
+1

**Pull request merged**